src/main/resources/static/overview.js

```diff
@@ -23,7 +23,7 @@
  */
 var HEADER_CONTENT = 'Black Duck Hub Security Analysis';
 var PAGE_SIZE = 100;
-var METRIC_KEYS = 'num_vuln_low, num_vuln_med, num_vuln_high, hub_component_names, num_components_rating';
+var METRIC_KEYS = 'num_vuln_low, num_vuln_med, num_vuln_high,num_vuln_critical, hub_component_names, num_components_rating';
```

Comment:
I'm not sure if whitespace matters wherever this is used, but there is a space after every comma except before `num_vuln_critical`.